src/config.rs

```diff
@@ -780,13 +780,14 @@ impl DbConfig {
                 })
         }
 
-        if self.rate_bytes_per_sec.0 > 0 {
-            opts.set_ratelimiter_with_auto_tuned(
-                self.rate_bytes_per_sec.0 as i64,
-                self.rate_limiter_mode,
-                self.auto_tuned,
-            );
-        }
+        //if self.rate_bytes_per_sec.0 > 0 {
+        opts.set_ratelimiter_with_auto_tuned(
+            10*1024*1024 ,
```

Comment:
10MB is too small?

Reply:
It is the initial value only. The rate bytes per sec will be dynamically adjusted.